### PR TITLE
[DOCS] Add the "investigate in timeline" button to the alert prevalence section

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2065. 

Preview:

- [Manage detection alerts | Overview](https://security-docs_2077.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#view-alert-details): Added a sentence about clicking on the alert prevalence count to open it in Timeline. The screenshot of the updated Alert details flyout is in https://github.com/elastic/security-docs/pull/2067.

### CODE REVIEW

File: docs/detections/alerts-ui-manage.asciidoc
Comment (benironside) [inline docs/detections/alerts-ui-manage.asciidoc:112]:
```suggestion
The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for the `host.name` field means three alerts with the same `host.name` value exist within the given timeframe. Alert prevalence data can help you investigate relationships with other alerts and gain more context about the event producing the alert. Click the alert prevalence count to explore the alerts in Timeline. 
```

**Review by benironside — COMMENTED** [review]
Looks good, left one minor suggestion.

File: docs/detections/alerts-ui-manage.asciidoc
Comment (joepeeples) [inline docs/detections/alerts-ui-manage.asciidoc:112]: Some minor suggestions for conciseness and flow. Adding the new sentence at the end made the paragraph feel a little long, so maybe breaking into two paras could help:
```suggestion
The *Alert prevalence* column shows the total number of alerts within the selected timeframe that have identical values. For example, an alert prevalence of 3 for `host.name` means three alerts with the same `host.name` value exist within the timeframe. 

Alert prevalence data can help you investigate relationships with other alerts and gain context about the events producing alerts. You can also click the alert prevalence count to explore the alerts in Timeline. 
```

**Review by joepeeples — APPROVED** [review]
LGTM! I added some totally optional suggestions for tightening up the paragraph.

File: docs/detections/alerts-ui-manage.asciidoc
Comment (nastasha-solomon) [reply docs/detections/alerts-ui-manage.asciidoc:112]: I'm on board with this. I usually try to keep all related content together, but I also like the idea of breaking paragraphs into 2-3 sentences when the sentences are on the longer side.

**Review by michaelolo24 — APPROVED** [review]
🚀 LGTM!

**Review by janmonschke — APPROVED** [review]

**Review by jmikell821 — APPROVED** [review]
LGTM - thanks for merging in feedback :-) We'll wait for QA to do a final review.

**Comment by karanbirsingh-qasource** [comment]
https://github.com/elastic/security-docs/pull/2077


Hi @nastasha-solomon 

we have validated this preview links for the respected changes and all changes are available and good to go.

**Fixed Areas:**

- [Documentation for Investigation in timeline for alert prevalence](https://security-docs_2077.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#view-alert-details) ✔️ 

image

- [Alert Flyout > Alert prevalence field have hyperlinks](https://security-docs_2067.docs-preview.app.elstc.co/guide/en/security/master/alerts-ui-manage.html#view-alert-details) ✔️ 

image

**Pull request merged**
